examples/nextjs/next.config.js

```diff
@@ -3,7 +3,7 @@ const nextConfig = {
   reactStrictMode: true,
   // Until the @apollo-client fixes the ESM modules support (https://github.com/apollographql/apollo-feature-requests/issues/287)
   // it's required to either transpile the `@lens-protocol` packages or make sure they won't get `imported` during SSR.
-  transpilePackages: ['@lens-protocol/react', '@lens-protocol/wagmi'],
+  transpilePackages: ['@lens-protocol'],
```

Comment:
this makes sure it works properly with `flat` `node_modules` (.e.g npm package manager)